pkg/kube/util.go

```diff
@@ -144,12 +143,6 @@ func SetRestDefaults(config *rest.Config) *rest.Config {
 	if len(config.ContentType) == 0 {
 		config.ContentType = runtime.ContentTypeJSON
 	}
-	if config.NegotiatedSerializer == nil {
-		// This codec factory ensures the resources are not converted. Therefore, resources
-		// will not be round-tripped through internal versions. Defaulting does not happen
-		// on the client.
-		config.NegotiatedSerializer = scheme.Codecs.WithoutConversion()
```

Comment:
This is not the right setup, we should set each groups' sheme.

But this field is also a must.